near-plugins-derive/src/pausable.rs

```diff
@@ -169,28 +176,16 @@ pub fn if_paused(attrs: TokenStream, item: TokenStream) -> TokenStream {
 }
 
 fn get_bypass_condition(args: &ExceptSubArgs) -> proc_macro2::TokenStream {
-    let self_condition = if args._self {
-        quote!(
-            if ::near_sdk::env::predecessor_account_id() == ::near_sdk::env::current_account_id() {
-                __check_paused = false;
-            }
-        )
-    } else {
-        quote!()
-    };
-
-    let owner_condition = if args.owner {
-        quote!(
-            if Some(::near_sdk::env::predecessor_account_id()) == self.owner_get() {
-                __check_paused = false;
-            }
-        )
-    } else {
-        quote!()
-    };
-
+    let except_roles = args.roles.clone();
     quote!(
-        #self_condition
-        #owner_condition
+        let __except_roles: Vec<&str> = vec![#(#except_roles.into()),*];
+        let __except_roles: Vec<String> = __except_roles.iter().map(|&x| x.into()).collect();
+        let may_bypass = self.acl_has_any_role(
+            __except_roles,
```

Comment:
I wonder if we could avoid a clone here by changing what type `acl_has_any_role` accepts.

Reply:
I've modified #38 to include that point as well.